ui/src/components/CustomControl/CustomControl.tsx

```diff
@@ -16,10 +16,10 @@ interface IData {
 interface ICustomCompClass {
     new (
         config: GlobalConfig,
+        el: HTMLElement | undefined,
```

Comment:
The docs does not say it is optional. If it would be undefined, it breaks the code in custom components

Reply:
yea, in the end it never is undefined, just at the very beginning before ref is assigned. We need to refactor this part  of code, but lets make it in other PR.